server/liveblog/themes/themes_assets/default/js/theme/handlers.js

```diff
@@ -84,15 +83,15 @@ var buttons = {
     },
 
     "[data-js-orderby_ascending]": () => {
-      loadSort('ascending');
+      if(isOrderChanged('ascending')) loadSort('ascending');
     },
 
     "[data-js-orderby_descending]": () => {
-      loadSort('descending');
+      if(isOrderChanged('descending')) loadSort('descending');
     },
 
     "[data-js-orderby_editorial]": () => {
-      loadSort('editorial');
+      if(isOrderChanged('editorial')) loadSort('editorial');
```

Comment:
I'm not sure about this. The conditions don't make sense. Would it be better to have the condition checked inside `loadSort` method?

Reply:
Yes, will move it to loadSort.